draft-ietf-quic-recovery.md

```diff
@@ -873,10 +873,9 @@ max_ack_delay:
   received ACK frame may be larger due to late timers, reordering,
   or lost ACKs.
 
-loss_time:
-: The time at which the next packet will be considered lost based on
-  exceeding the reordering window in time. Only applies to the ApplicationData
-  packet number space.
+loss_times[kPacketNumberSpace]:
```

Comment:
Nit: `loss_time` would be more consistent, since we also use `largest_acked_packet[kPacketNumberSpace]`, not `largest_acked_packets[kPacketNumberSpace]`.